pkg/webhooks/admission/hypernodes/validate/admit_hypernode.go

```diff
@@ -121,6 +148,15 @@ func validateHyperNodeMemberSelector(selector hypernodev1alpha1.MemberSelector,
 			errs = append(errs, err)
 		}
 	}
+	if selector.LabelMatch != nil {
+		if (selector.LabelMatch.MatchExpressions == nil || len(selector.LabelMatch.MatchExpressions) == 1) &&
```

Comment:
What are these codes for? If matchExpressions and matchLabels are both be empty, it means matching all nodes. And we didn't add validation in kubebuilder validation format, I don't think we need these codes